server/controllers/routes/user/index.js

```diff
@@ -3,7 +3,9 @@ const router = require('express').Router();
 const { getAlumni } = require('./student');
 const { getProjectData, getProjectsData } = require('./project');
 const { getCohortsData, getSpecificCohort } = require('./cohort');
+const { getAlumniCohort } = require('./student');
 
+router.get('/alumni/cohorts/:cohortId', getAlumniCohort);
```

Comment:
I think this route would be better if was like this `/cohorts/:cohortId/alumni`
this way you can read it as `get the alumni of this cohort`
what do you think?

Reply:
yes you are right there is no needed of this query if cohortId doesn't exist  it will send an empty array.
![58745555](https://user-images.githubusercontent.com/55782435/78453309-80ba7a80-7699-11ea-972f-e324be2a473a.png)
i will modify it, thank you :100:

Reply:
yes it seems better